requirements.txt

```diff
@@ -2,11 +2,12 @@ boto3
 botocore
 flask
 flask-jwt-extended
-flask-ldap3-login
 flask-restx
 python-dotenv
 redis
 uwsgi
 requests
 mock==4.0.2
```

Comment:
version constraints make baby Jesus cry 😉

Reply:
oh yes - that is a moto problem. I spent many hours on it ...